.travis.yml

```diff
@@ -1,8 +1,10 @@
----
```

Comment:
yaml-lint complains if there's no document start marker, so I'd put that back in.